src/EPR.Calculator.API.UnitTests/Builder/CalcResultBuilderTests.cs

```diff
@@ -18,41 +20,41 @@ public class CalcResultBuilderTests
     {
         private Mock<ICalcResultDetailBuilder> calcResultDetailBuilder;
         private Mock<ICalcResultLapcapDataBuilder> lapcapBuilder;
-        private Mock<ICalcResultOnePlusFourApportionmentBuilder> mockOnePlusFourApportionmentBuilder;
-        private Mock<ICalcResultParameterOtherCostBuilder> mockICalcResultParameterOtherCostBuilder;
+        private Mock<ICalcResultLateReportingBuilder> lateReportingBuilder;
+        private Mock<ICalcRunLaDisposalCostBuilder> runLaDisposalCostBuilder;
         private Mock<ICalcResultCommsCostBuilder> commsCostReportBuilder;
-        private Mock<ICalcResultLateReportingBuilder> mockLateReportingBuilder;
-        private Mock<ICalcResultParameterOtherCostBuilder> mockCalcResultParameterOtherCostBuilder;
-        private Mock<ICalcRunLaDisposalCostBuilder> mockRunLaDisposalCostBuilder;
+        private Mock<ICalcResultParameterOtherCostBuilder> calcResultParameterOtherCostBuilder;
+        private Mock<ICalcResultOnePlusFourApportionmentBuilder> onePlusFourApportionmentBuilder;
+        private Mock<ICalcResultSummaryBuilder> summaryBuilder;
 
         [TestInitialize]
         public void SetUp()
         {
             this.calcResultDetailBuilder = new Mock<ICalcResultDetailBuilder>();
             this.lapcapBuilder = new Mock<ICalcResultLapcapDataBuilder>();
-            this.mockICalcResultParameterOtherCostBuilder = new Mock<ICalcResultParameterOtherCostBuilder>();
-            this.mockOnePlusFourApportionmentBuilder=new Mock<ICalcResultOnePlusFourApportionmentBuilder> ();
+            this.onePlusFourApportionmentBuilder=new Mock<ICalcResultOnePlusFourApportionmentBuilder> ();
             this.commsCostReportBuilder = new Mock<ICalcResultCommsCostBuilder>();
-            this.mockLateReportingBuilder = new Mock<ICalcResultLateReportingBuilder>();
-            this.mockCalcResultParameterOtherCostBuilder = new Mock<ICalcResultParameterOtherCostBuilder>();
-            this.mockRunLaDisposalCostBuilder = new Mock<ICalcRunLaDisposalCostBuilder>();
+            this.lateReportingBuilder = new Mock<ICalcResultLateReportingBuilder>();
+            this.calcResultParameterOtherCostBuilder = new Mock<ICalcResultParameterOtherCostBuilder>();
+            this.runLaDisposalCostBuilder = new Mock<ICalcRunLaDisposalCostBuilder>();
         }
 
-        [TestMethod]
-        public void CanConstruct()
-        {
-            // Act
-            var instance = new CalcResultBuilder(
-                calcResultDetailBuilder.Object,
-                lapcapBuilder.Object,
-                mockICalcResultParameterOtherCostBuilder.Object,
-                mockOnePlusFourApportionmentBuilder.Object,
-                commsCostReportBuilder.Object,
-                mockLateReportingBuilder.Object,
-                mockRunLaDisposalCostBuilder.Object);
```

Comment:
Why is this commented?

Reply:
Good catch @udaydefra, thank you. I have uncommented the tests.